modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc

```diff
@@ -45,6 +45,20 @@ Mismatch for thumbnails/9/2048-2048-max.png:
  Actual:  SHA1:da39a3ee5e6b4b0d3255bfef95601890afd80709 MD5:d41d8cd98f00b204e9800998ecf8427e ADLER32:00000001
 ----
 
+=== Options
+
+[width="100%",cols="20%,70%",]
+|===
+|`-r, --repair`         
+|Repair filecache-entry with mismatched checksums.
+
+|`-u, --user=USER`       
+|Specific user to check.
+
+|`-p, --path=PATH`       
+|Path to check relative to user folder, i.e, relative to /john/files. e.g tree/apple [default: ""].
```

Comment:
You may want to look to https://doc.owncloud.org/server/10.4/admin_manual/configuration/server/occ_command.html#the-path-option
for examples which have better readability

Reply:
Thank you for the suggestion.

Reply:
Just extended the description from the one provided by the command itself.